src/main/java/slimeknights/tconstruct/library/tinkering/IAoeTool.java

```diff
@@ -22,13 +23,18 @@
    * @param origin the origin block spot to start from
    * @return A list of BlockPoses that the AOE tool can affect.
    */
-  ImmutableList<BlockPos> getAOEBlocks(@Nonnull ItemStack stack, World world, PlayerEntity player, BlockPos origin);
+  default ImmutableList<BlockPos> getAOEBlocks(@Nonnull ItemStack stack, World world, PlayerEntity player, BlockPos origin) {
+    return AoeToolInteractionUtil.calculateAOEBlocks(stack, world, player, origin, 1, 1, 1);
+  }
 
 
   /**
    * Checks if the AOE tool mines blocks, used for the extra block breaking rendering.
    *
    * @return if the tool is a AOE harvest tool
    */
-  boolean isAoeHarvestTool();
+  default boolean isAoeHarvestTool() {
```

Comment:
I still think we should get rid of this method entirely for now. If we do keep this or something similar, we can add it later using a parameter saying what you want, so something can be AOE harvest, AOE interact, etc. I think we have tool types somewhere that would help with that

But anyways, for now lets ditch this method.

Reply:
Has been removed 👍